src/devtools/client/debugger/src/actions/breakpoints/index.js

```diff
@@ -228,3 +225,17 @@ export function removeBreakpointsAtLine(cx, sourceId, line) {
     return dispatch(removeBreakpoints(cx, breakpointsAtLine));
   };
 }
+
+function getLogValue(source, state, location) {
+  const file = getFilename(source);
+  const symbols = getSymbols(state, source);
+  const { line, column } = location;
+
+  if (!column) {
+    const closestSymbol = findClosestEnclosedSymbol(symbols, { line });
+    return closestSymbol ? `"${closestSymbol.name}"` : `"${file}:${line}"`;
+  } else {
+    const closestSymbol = findClosestEnclosedSymbol(symbols, location);
+    return closestSymbol ? `"${closestSymbol.name}"` : `"${file}:${line}:${column}"`;
+  }
+}
```

Comment:
Small code tweak

```js
const symbol = findClosestEnclosedSymbol(symbols, location);

if (symbol) {
  return symbol.name
}

let logValue = `"${file}:${line}"`
if (column) {
  logValue += `:${column}`
}

return logValue
```

Reply:
it does, i tested breakpoints in different files and column breakpoints as well, also checked the breakpoint names in the console and breakpoints panel in the sidebar. i'll make the tweak today.